e2e/e2eutils/setup.go

```diff
@@ -69,8 +69,7 @@ func MakeDeployParams(t require.TestingT, tp *TestParams) *DeployParams {
 		ValidatorPoolTrustedValidator: addresses.TrustedValidator,
 		ValidatorPoolMinBondAmount:    uint64ToBig(1),
 		ValidatorPoolMaxUnbond:        10,
-		ValidatorPoolNonPenaltyPeriod: 2,
-		ValidatorPoolPenaltyPeriod:    2,
+		ValidatorPoolRoundDuration:    6,
```

Comment:
It probably should be 4 because the `L2OutputOracleSubmissionInterval` is 4 as below